### PR TITLE
chore(deps): bump Angular to 14.2 from 14.0 to address several CVEs

### PR DESCRIPTION
**Pull request by joanise**
These Dependabot alerts are all fixed by upgrading Angular to 14.2 and re-creating package-lock.json from scratch:
 - https://github.com/ReadAlongs/Studio-Web/security/dependabot/2
 - https://github.com/ReadAlongs/Studio-Web/security/dependabot/4
 - https://github.com/ReadAlongs/Studio-Web/security/dependabot/5
 - https://github.com/ReadAlongs/Studio-Web/security/dependabot/8

Tested on my machine with the automated test suite and manual testing.

CI passes too, on Node 14, 16 and 18.

### CODE REVIEW

**Comment by joanise** [comment]
@roedoejet @dhdaines @deltork This PR will require rerunning `npm install` when you update. But Angular 14.2 seems to update a bunch of packages for CVEs without changing any functionality. I've already moved Web-Component/packages/angular-component to Angular 14.2 since it is not in active use yet, but I want to give you a warning before updating this one, via this PR.

**Comment by dhdaines** [comment]
This seems good to me!  Thanks for the unit test example!

**Comment by joanise** [comment]
@dhdaines Thanks! Ready to merge, then?

File: src/app/upload/upload.component.html [hunk 1]
```diff
@@ -35,10 +35,10 @@ <h1 class="title">Text</h1>
                 </div>
                 <div *ngIf="inputMethod.text === 'edit'">
                     <div class="row">
-                        <div class="col-10">
+                        <div class="col-6 col-md-8 col-lg-9">
```
Comment (roedoejet) [inline src/app/upload/upload.component.html]: Why change this? It now appears a bit offset from the right edge: 
Original: image

This PR: 
image

**Review by roedoejet — CHANGES_REQUESTED** [review]
All looks great to me, and gets rid of all the security stuff as you say. The only thing is you've changed around some of the grids, which fixes some things, like going from this:

image

to this: 

image

which is awesome! but it also has the save button offset from the right side as noted in a comment. Can you rix this? (might need some additional css. Thanks @joanise !

**Comment by dhdaines** [comment]
> @dhdaines Thanks! Ready to merge, then?

No problem for me, merge when you're ready (and see Aidan's comments above)

File: src/app/upload/upload.component.html (hunk 1 above)
Comment (joanise) [reply src/app/upload/upload.component.html]: On larger screens, the Save button is nicely centered under the Write and File buttons, but it's not so nice on smaller screens. I'll find better setting for smaller screens but I like my large screen view.
While I'm at it, I'll also try to fix this:
image

Comment (joanise) [reply src/app/upload/upload.component.html]: So I suggest keeping this:
image
but fixing this:
image
by aligning it to the right when the screen gets smaller.

**Comment by joanise** [comment]
@roedoejet updated and ready for re-reviewing.
md+:
image
md:
image
sm-:
image

**Comment by joanise** [comment]
And, for the record, I hate HTML, even in 2022 with all the frameworks and libraries. All this formatting nonsense, half of it not working at any given time, and never the same half from one day to the next. Bleh.

File: src/app/upload/upload.component.html (hunk 1 above)
Comment (joanise) [reply src/app/upload/upload.component.html]: And nevermind this, it's now always on the right.

**Comment by dhdaines** [comment]
> And, for the record, I hate HTML, even in 2022 with all the frameworks and libraries. All this formatting nonsense, half of it not working at any given time, and never the same half from one day to the next. Bleh.

It's really CSS that's the problem... decades later you still can't reliably do certain types of layouts that are easy to achieve with tables, and the whole point of CSS layouts was supposed to be to avoid using markup for presentation purposes...

**Comment by joanise** [comment]
> > And, for the record, I hate HTML, even in 2022 with all the frameworks and libraries. All this formatting nonsense, half of it not working at any given time, and never the same half from one day to the next. Bleh.
> 
> It's really CSS that's the problem... decades later you still can't reliably do certain types of layouts that are easy to achieve with tables, and the whole point of CSS layouts was supposed to be to avoid using markup for presentation purposes...

That's a fair point, all the stuff I'm fighting with is either classes which trigger css, or `style=""` which is just more css inlined. In the Coursera course too, I always find it annoying how to instructor adds this and that class and you're just supposed to know why. I have to experiment quite a bit to make sense of stuff.

Things like bootstrap or fxFlex layouts make things easier, for sure, but there just continue to be annoying corner cases where I feel the code is fighting me!

**Pull request merged**